src/components/Dropdown.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+
+export const Dropdown = ({onFilter}) => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const handleFilter = (region) => {
+        onFilter(region);
+        setIsOpen(false);
+    };
+
+    return (
+        <div className={`dropdown-wrapper ${isOpen ? "open" : ""}`}>
+            <div
+                className="dropdown-header flex flex-jc-sb flex-ai-c"
+                onClick={() => setIsOpen(!isOpen)}
+            >
+                <span>Filter by Region</span>
+                <i className="fa-regular fa-chevron-down icon"></i>
+            </div>
+            {isOpen && (
+                <div className="dropdown-body">
+                    <ul>
+                        {["All", "Africa", "Americas", "Asia", "Europe", "Oceania"].map(
+                            (region) => (
```

Comment:
Good approach, but you can save the array inside constants folder and not hardcode it like this